.github/workflows/PR_reviewer_test.yml

```
@@ -1,24 +1,59 @@
+          node-version: '20'
 
-      - name: Run PR Reviewer Script
+      - name: Install Dependencies
         run: |
-          node firefliesai/.github/.github/scripts/PR_reviewer.js
\ No newline at end of file
+          cd .github/scripts/pr-review
+          npm install
+
+      - name: Run PR Reviewer
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)